pkg/resource/job.go

```diff
@@ -140,8 +140,8 @@ func (b JobBuilder) MakeContainers() []corev1.Container {
 			ImagePullPolicy: b.GetImagePullPolicy(),
 			Resources: corev1.ResourceRequirements{
 				Limits: corev1.ResourceList{
-					corev1.ResourceCPU:    apiresource.MustParse("300m"),
-					corev1.ResourceMemory: apiresource.MustParse("256Mi"),
+					corev1.ResourceCPU:    apiresource.MustParse("500m"),
```

Comment:
Could we defer from updating the CPU limit unless required?

Reply:
Done